feature-libs/product-configurator/rulebased/components/attribute/types/numeric-input-field/configurator-attribute-numeric-input-field.component.ts

```diff
@@ -117,29 +96,12 @@ export class ConfiguratorAttributeNumericInputFieldComponent
       .subscribe(() => this.onChange());
   }
 
-  /**
-   * Hit when user input was changed
-   */
   onChange(): void {
```

Comment:
can we just remove this `onChange`?

Reply:
Do I understand you correctly if `onChange` is removed here then `onChange` from `ConfiguratorAttributeInputFieldComponent` will be called when necessery?

Reply:
I think so. Can you do some tests of `onChange` for `numeric-input-field`?

Reply:
Done